Bottom: 49cb6aa3d3dee206ef9cbdc19ec9e15dcf17685d
Top:    71323f790daba0f5d4e67f91c319beb123273fbd
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-23 17:43:30 +0800

Refresh of vhost-introduce-helper-to

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index a8b5488..dc8c3d9 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2502,6 +2502,7 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 			vq_err(vq, "Failed to get descriptors\n");
 			return -EFAULT;
 		}
+		vhost_add_used_elem(vq, indices[0], 0, ret - total);
 		--total;
 		++indices;
 		++last_avail_idx;
